File: 10-1.cpp
#include "opencv2/opencv.hpp"   

using namespace cv;

int main(){

    Mat img;
    img = imread("test.png");

    Mat kernel = Mat(5,5, CV_32F, Scalar(1/25.0));

    Mat dst;
    filter2D(img, dst, -1, kernel);

    imshow("Original", img);
    imshow("result", dst);

    waitKey(0);
    destroyAllWindows();
    return 0;
}

File: 10-1.py
import cv2 as cv
import numpy as np
from sympy import im

img = cv.imread('test.png')

kernel = np.ones((5,5), np.float32)/25

dst = cv.filter2D(img, -1, kernel)

cv.imshow('Original', img)
cv.imshow('Result', dst)

cv.waitKey(0)
cv.destroyAllWindows()